java/arcs/android/sdk/host/ArcHostService.kt

```diff
@@ -10,6 +10,7 @@
  */
 package arcs.android.sdk.host
 
+import android.app.Service
```

Comment:
Unused import?

Reply:
I was confused that ktlint didn't fire.
Looks like this got imported for the doc reference...